### PR TITLE
Fix one pod to multiple PVC connection

### PR DESCRIPTION
**Pull request by Akash4927**
Read all the PVC source from a pod spec
and add adjacency between the pod and its PVCs.

screenshot from 2019-01-09 16-25-07

Signed-off-by: Akash Srivastava <akashsrivastava4927@gmail.com>

fixes: #3550

### CODE REVIEW

File: probe/kubernetes/pod.go [hunk 1]
```diff
@@ -94,8 +95,8 @@ func (p *pod) GetNode(probeID string) report.Node {
 		RestartCount:          strconv.FormatUint(uint64(p.RestartCount()), 10),
 	}
 
-	if p.VolumeClaimName() != "" {
-		latests[VolumeClaim] = p.VolumeClaimName()
+	if len(p.VolumeClaimNames()) > 0 {
+		latests[VolumeClaim] = strings.Join(p.VolumeClaimNames(), "~p$")
```
Comment (bboreham) [inline probe/kubernetes/pod.go]: `"~p$"` should be a constant.
And is that necessary and/or sufficient?  What are the rules on volume claim names?

Comment (Akash4927) [reply probe/kubernetes/pod.go]: PVC name must consist of lower case alphanumeric characters, `-` or `.`, and must start and end with an alphanumeric character.
We can use a single space or any special character to join them.

Comment (bboreham) [reply probe/kubernetes/pod.go]: OK, can you please add a comment about the rules and put the separator in a `const`.
I would probably just use `,` as the separator, if that works, but don't really mind.

Comment (Akash4927) [reply probe/kubernetes/pod.go]: Sure @bboreham, `,` will work.
I'll update the PR soon.

**Pull request merged**